doordeck-sdk/src/commonMain/kotlin/com/doordeck/multiplatform/sdk/internal/api/AccountlessClient.kt

```diff
@@ -18,9 +20,12 @@ internal open class AccountlessClient(
      * @see <a href="https://developer.doordeck.com/docs/#login-v2">API Doc</a>
      */
     suspend fun loginRequest(email: String, password: String): TokenResponse {
-        return httpClient.post(Paths.getLoginPath()) {
+        return httpClient.post<TokenResponse>(Paths.getLoginPath()) {
             addRequestHeaders(apiVersion = ApiVersion.VERSION_2)
             setBody(LoginRequest(email, password))
+        }.also {
+            contextManager.setAuthToken(it.authToken)
+            contextManager.setRefreshToken(it.refreshToken)
```

Comment:
From now on, we will update the context values whenever possible. Previously, this had to be done manually by the developer. Please note that this only updates the variable; it does not persist it.